widgets/contact/contact.php

```diff
@@ -121,6 +121,11 @@ function get_widget_form() {
 							'required_fields[hide]' => array( 'hide' ),
 						)
 					),
+					'log_ip_address' => array(
```

Comment:
I assume this defaults to false either way, but we should have a `'default' => false` just to clarify.

Reply:
👍